app/views/volunteers/index.html.erb

```diff
@@ -30,21 +30,23 @@
           </a>
         </li>
         <% current_organization.supervisors.each do |supervisor| %>
```

Comment:
You could filter for active supervisors here insttead of adding a check below
`current_organization.supervisors.where(active: true). each do`...

Reply:
That is a nice suggestion, I will push it in another PR :)